### PR TITLE
Support for arm64

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

File: azure-iot-edge-archetype/src/main/resources/archetype-resources/Dockerfile.arm64v8 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM arm64v8/maven:3.5-jdk-8-slim AS build-env
+RUN mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.1:get -Dartifact=com.microsoft.azure.sdk.iot:iot-device-client:1.14.0 -DrepoUrl=https://repo.maven.apache.org/maven2/
```
Comment (adashen) [inline azure-iot-edge-archetype/src/main/resources/archetype-resources/Dockerfile.arm64v8]: latest java sdk is 1.17.5, how about upgrade the sdk version also? This also need to update https://github.com/microsoft/azure-maven-archetypes/blob/develop/azure-iot-edge-archetype/src/main/resources/archetype-resources/pom.xml#L17

Comment (formulahendry) [reply azure-iot-edge-archetype/src/main/resources/archetype-resources/Dockerfile.arm64v8]: updated

File: azure-iot-edge-archetype/pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-iot-edge-archetype</artifactId>
-    <version>1.3.0-SNAPSHOT</version>
+    <version>1.3.0-arm64-preview</version>
```
Comment (adashen) [inline azure-iot-edge-archetype/pom.xml]: 1.4.0-arm64-preview?

Comment (formulahendry) [reply azure-iot-edge-archetype/pom.xml]: The latest version is 1.2.0: https://mvnrepository.com/artifact/com.microsoft.azure/azure-iot-edge-archetype. So I think it should be 1.3.0-arm64-preview

Comment (adashen) [reply azure-iot-edge-archetype/pom.xml]: got it

**Pull request merged**